_posts/dev/2014-08-02-kura-dio-apis.md

```diff
@@ -128,6 +130,205 @@ button.setInputListener(new PinListener(){
 		}
 });
 
+```
+### Accessing a GPIO Pin with GPIOService for a switch a Led. 
+
```

Comment:
Probably you need to specify that this example is specifically crafted for a raspberry pi. It will probably work with other platforms by changing the pins used.